tests/PySys/call_tedge_connect_c8y_--test_negative/run.py

```diff
@@ -0,0 +1,31 @@
+from pysys.basetest import BaseTest
+
+"""
+Test `tedge connect c8y --test` (successful case):
```

Comment:
Better change that line to
"Run connection test without being connected (negative case):"